.azurepipelines/publish-maven-1ES.yml

```diff
@@ -0,0 +1,115 @@
+trigger: none
+name: $(date:yyyyMMdd)$(rev:.r)
+variables:
+- name: BUILD_DIR
+  value: $(Build.ArtifactStagingDirectory)
+- group: AppCenter-SDK-Android Bintray and Maven
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+extends:
+  ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/master') }}:
+    template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  ${{ else }}:
+    template: v1/1ES.Unofficial.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: Azure Pipelines
+      image: macos-latest
+      os: macOS
+    customBuildTags:
+    - ES365AIMigrationTooling-BulkMigrated
+    sdl:
+      sourceAnalysisPool: 1ES-PT-Windows-2022
+    stages:
+    - stage: stage
+      jobs:
+      - job: Phase_1
+        displayName: MacOS
+        cancelTimeoutInMinutes: 1
+        templateContext:
+          outputs:
+          - output: pipelineArtifact
+            displayName: 'Publish Artifact: Release'
+            targetPath: $(Build.ArtifactStagingDirectory)/com/microsoft
+            artifactName: Release
+        steps:
+        - checkout: self
+          clean: true
+          submodules: recursive
+          fetchTags: false
+        - task: JavaToolInstaller@0
+          displayName: Use Java 11
+          inputs:
+            versionSpec: 11
+            jdkArchitectureOption: x64
+            jdkSourceOption: PreInstalled
+        - task: DownloadSecureFile@1
+          displayName: Download GPG-key
+          inputs:
+            secureFile: 98b107ca-fab1-42c0-819d-2871c022869c
+        - task: Bash@3
+          displayName: Put Azure Credentials
+          inputs:
+            filePath: ./scripts/put-azure-credentials.sh
+            arguments: $(AZURE_USERNAME) $(AZURE_PASSWORD)
+        - task: ShellScript@2
+          displayName: Configure bintray
+          inputs:
+            scriptPath: scripts/put-bintray-secrets-gradle.sh
+            args: $(MAVEN_USER) $(MAVEN_KEY) $(GDP_SIGNING_KEY_ID) "$(Agent.TempDirectory)/appcenter-gpg-key.gpg" $(GDP_KEY_PASSWORD)
+            disableAutoCwd: true
+        - task: Bash@3
+          displayName: Set NDK version
+          inputs:
+            filePath: ./scripts/set-ndk-version.sh
+        - task: Gradle@1
+          displayName: Gradle publish to Maven local
+          inputs:
+            tasks: publish
+            publishJUnitResults: false
+        - task: SFP.release-tasks.custom-build-release-task.EsrpRelease@4
+          displayName: ESRP Release
+          inputs:
+            ConnectedServiceName: ESRP Release
+            FolderLocation: $(Build.ArtifactStagingDirectory)/com/microsoft
+            Owners: lucen@microsoft.com
+            Approvers: lucen@microsoft.com

```

Comment:
Should it be a single person who can approve? Can it be changed to some group of approvers?

Reply:
I'm not sure but let's try. Is there some declared group that we could try to specify? Or we could try to use your email?